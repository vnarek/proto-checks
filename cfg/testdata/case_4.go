package main

//infinite loop
func main() {
	x = *y
	// var x = *y // hier
	x := *y
}
