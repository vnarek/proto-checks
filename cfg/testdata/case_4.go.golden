[START]
  [x = *y]
    [x = *y]
